### DISCUSSION TITLE
nargo info circuit size

### DISCUSSION THREAD

**Discussion by liudi4046** [noir-lang/noir#5713] [Q&A] [answered]
Is the circuit size printed by `nargo info` correspond to Barretenberg backend?

**Comment by Savio-Sou** [comment, marked as answer]
From Noir v0.31.0 onwards, you can measure the Barretenberg gate counts of Noir programs using the `bb gates` command:
https://github.com/AztecProtocol/aztec-packages/blob/7065962bb507555bcbb25cb3bbfc2a0a90687000/barretenberg/cpp/src/barretenberg/bb/main.cpp#L1427
